docs/legacy_installation.md

```diff
@@ -0,0 +1,56 @@
+### Legacy installation (without Symfony Flex)
+
+1. Require plugin with composer:
+
+    ```bash
+    composer require sylius/admin-order-creation-plugin
+    ```
+
+2. Import configuration:
```

Comment:
Could we provide which file should be modified? `app/config/config.yml` it this case,